tasks/redhat.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: rhel | Installing DRBD packages
+  yum:
+    name: "{{ item }}"
+    state: present
+  become: true
+  with_items: "{{ drbd_additional_packages }}"
```

Comment:
We could change this to `drbd_redhat_additional_packages` for Debian based apt packages. This would allow the flexibility to not have to define this variable differently between Debian and RHEL based OS's.